pkg/daemon/controller/node.go

```diff
@@ -79,25 +84,34 @@ func (nic *NodeIPCache) UpdateNodeIPs(nodeList []*v1.Node, localNodeName string)
 		}
 	}
 
+	for _, remoteVtep := range remoteVtepList {
```

Comment:
do we need feature-gate validation here?

Reply:
Maybe not, because if multicluster is enabled, `remoteVtepList` will be empty.